scripts/integration/opentelemetry/test.yaml

```diff
@@ -2,11 +2,12 @@ args:
 - --features
 - opentelemetry-integration-tests
 - --lib
-- 'sources::opentelemetry::integration_tests::'
+- '::opentelemetry::'
```

Comment:
**Question**: this is equivalent, just shorter?

Reply:
If we set up an otel sink or other otel components, this integration won't have to be changed to run those tests as well.

Reply:
Ah, duh - I keep forgetting that we don't necessarily care about splitting up source and sink integration tests